source/llvm/build-source-tarball.sh

```diff
@@ -68,6 +68,7 @@ function build_llvm() {
   wrap cmake ${THIS_DIR}/$LLVM.src \
       -DCMAKE_BUILD_TYPE=Release \
       -DCMAKE_INSTALL_PREFIX=$LOCAL_INSTALL \
+      -DGCC_INSTALL_PREFIX=$GCC_INSTALL_PREFIX \
```

Comment:
BTW, if we want to use this across several of the llvm scripts, maybe we can put the new variable in source/llvm/build.sh since I think that one calls all of the others.